python/test/test_comb.py

```diff
@@ -88,5 +96,5 @@ def test_cast_zcdp_approxdp():
     print(smd_gaussian.map(1.).epsilon(1e-6))
     
 if __name__ == "__main__":
-    test_cast_zcdp_approxdp()
+    test_make_basic_composition_approx()
 
```

Comment:
Leftovers?

Reply:
It's guarded by an if, so it only runs if you run the file manually. Seems fine to me if it changes over time